convert2rhel/subscription.py

```diff
@@ -878,9 +879,15 @@ def check_needed_repos_availability(repo_ids_needed):
                 " RHEL packages when converting. The converted system will end up"
                 " with a mixture of packages from RHEL and your current distribution." % repo_id
             )
+            non_avail_repos.append(repo_id)
             all_repos_avail = False
+
     if all_repos_avail:
         loggerinst.info("Needed RHEL repositories are available.")
+    if non_avail_repos:
+        return non_avail_repos
+    else:
+        return None
```

Comment:
Maybe we could just simplify this and return `non_avail_repos`

```suggestion
    return non_avail_repos
```

In whatever function that calls this, the checking condition would still be the same, like `if not ....`

Reply:
Noted i'll add this